packages/react/src/utils/productionContextError.ts

```diff
@@ -0,0 +1,3 @@
+export function productionContextError(): never {
+  throw new Error('Base UI: Invalid component tree');
+}
```

Comment:
React's minified errors link to a webpage that details the error, but this seems unnecessary for invalid component trees since it's certain to be encountered while developing beforehand